showcase/src/app/material-icons.config.ts

```diff
@@ -0,0 +1,14 @@
+import { MatIconRegistry } from "@angular/material";
+import { DomSanitizer } from "@angular/platform-browser";
+
+/**
+ * Registers the @mdi svg icon set to the Angular Material icon registry in the default namespace
+ * @param matIconRegistry - Angular Material's icon registry
+ * @param domSanitizer - Angular's DOM sanitizer
+ */
+export function registerMaterialIconSet(matIconRegistry: MatIconRegistry, domSanitizer: DomSanitizer): void {
+	// IMPORTANT: the path of the @mdi svg icon set should match the output path given to it in the angular.json file
+	const iconSetUrl: string = "assets/material-icons/mdi.svg";
+
+	matIconRegistry.addSvgIconSet(domSanitizer.bypassSecurityTrustResourceUrl(iconSetUrl));
```

Comment:
What do you think of providing this file directly in Stark-UI?

Reply:
The same reason I explained above...

Reply:
But that means our components won't be displayed correctly if they don't have the needed icons... :confused:
We should be careful about that

Reply:
Indeed, I understand but in this case we cannot enforce our icons (mdi icons) since it is a big file and in case they don't want those icons but others we will still copy our icons to the /assets folder.

In fact, the only requirement we have for the stark components to work is that there should be an icon with the same id in the icon set they provide. Therefore I think the most practical solution will be to leave that open to the user and just require them to register the icon set themselves, even if is the mdi icon set shipped in stark-ui.